include/deal.II/base/tensor.h

```diff
@@ -1362,8 +1362,9 @@ template <typename OtherNumber>
 DEAL_II_CONSTEXPR inline DEAL_II_CUDA_HOST_DEV Tensor<rank_, dim, Number> &
 Tensor<rank_, dim, Number>::operator/=(const OtherNumber &s)
 {
+  OtherNumber inverse_factor = OtherNumber(1.) / s;
```

Comment:
With the same argument as above, the right hand side should then be `Number(1.0) / s`.

Reply:
Drive by comment: If this causes problems with the AD tests, then the fix will probably be to use `internal::NumberType<OtherNumber>::value(1.)` here and below, and similarly in the point class. 

Also, this should be `const OtherNumber inverse_factor` :-)